projects/packages/my-jetpack/_inc/components/my-jetpack-tab-panel/products/mappings.ts

```diff
@@ -18,29 +18,44 @@ export const CATEGORY_CARDS_AND_MODULES: {
 } = {
 	security: {
 		cards: [ 'backup', 'scan', 'anti-spam' ],
-		modules: [ 'waf', 'protect', 'account-protection', 'monitor' ],
+		modules: [
+			'account-protection',
+			'monitor',
+			'notes',
+			'protect',
+			'sso',
+			'verification-tools',
+			'waf',
+		],
 	},
 	growth: {
-		cards: [ 'stats', 'social' ],
-		modules: [ 'blaze', 'related-posts', 'subscriptions', 'sharedaddy', 'seo-tools', 'wordads' ],
+		cards: [ 'stats', 'social', 'jetpack-ai', 'crm' ],
+		modules: [
+			'blaze',
+			'comment-likes',
+			'comments',
+			'contact-form',
+			'copy-post',
+			'likes',
+			'publicize',
+			'related-posts',
+			'seo-tools',
+			'sharedaddy',
+			'sitemaps',
+			'stats',
+			'subscriptions',
+			'woocommerce-analytics',
+			'wordads',
+		],
 	},
 	performance: {
-		cards: [ 'boost' ],
-		modules: [ 'search', 'photon', 'videopress' ],
-	},
-	management: {
-		cards: [ 'crm' ],
-		modules: [],
-	},
-	create: {
-		cards: [ 'ai' ],
-		modules: [ 'carousel', 'post-by-email' ],
+		cards: [ 'boost', 'search', 'videopress' ],
+		modules: [ 'photon', 'photon-cdn' ],
 	},
 	recommended: {
 		cards: [ 'stats', 'social' ],
 		modules: [
-			'stats',
-			'publicize',
+			// No prettier please
```

Comment:
Wait, doesn't prettier listen to that comment? that's cute 👀

Reply:
Without that comment, Prettier converts that to a single line which makes it difficult to sort.